run.sh

```diff
@@ -15,7 +15,8 @@ while true
 do
 cd $pokebotpath
 python pokecli.py -cf $filename
-read -p "Press any button or wait 20 seconds." -r -s -n1 -t 20
-echo `date`"Pokebot"$*" Stopped." 
+read -p "Press any button or wait 20 seconds to continue.
```

Comment:
What is the typo here? Moving those flags to the next line seems like a scary change.